code/modules/admin/verbs/striketeam.dm

```diff
@@ -1,31 +1,46 @@
-//STRIKE TEAMS
+//Deathsquad
 
-#define COMMANDOS_POSSIBLE 6 //if more Commandos are needed in the future
-GLOBAL_VAR_INIT(sent_strike_team, 0)
+#define MAX_COMMANDOS 14
+GLOBAL_VAR_INIT(strike_team_sent, FALSE)
 
 /client/proc/strike_team()
-	if(!SSticker)
-		to_chat(usr, "<span class='userdanger'>The game hasn't started yet!</span>")
+	var/proccaller = usr.client
+	if(!check_rights(R_EVENT))
 		return
-	if(GLOB.sent_strike_team == 1)
-		to_chat(usr, "<span class='userdanger'>CentComm is already sending a team.</span>")
-		return
-	if(alert("Do you want to send in the CentComm death squad? Once enabled, this is irreversible.",,"Yes","No")!="Yes")
+	if(SSticker.current_state == GAME_STATE_PREGAME)
+		to_chat(usr, "<span class='warning'>The round hasn't started yet!</span>")
 		return
+	if(GLOB.strike_team_sent)
+		if(alert("A Deathsquad is already being sent, are you sure you want to send another?",, "Yes", "No") != "Yes")
+			return
+	else
+		if(alert("Do you want to send in the Deathsquad? Once enabled, this is irreversible.",, "Yes", "No") != "Yes")
+			return
+	message_admins("<span class='notice'>[key_name_admin(proccaller)] has started to spawn a DeathSquad.</span>")
+	log_admin("[key_name_admin(proccaller)] has started to spawn a DeathSquad.")
 	alert("This 'mode' will go on until everyone is dead or the station is destroyed. You may also admin-call the evac shuttle when appropriate. Spawned commandos have internals cameras which are viewable through a monitor inside the Spec. Ops. Office. The first one selected/spawned will be the team leader.")
 
-	message_admins("<span class='notice'>[key_name_admin(usr)] has started to spawn a CentComm DeathSquad.</span>", 1)
-
-	var/input = null
-	while(!input)
-		input = sanitize(copytext(input(src, "Please specify which mission the death commando squad shall undertake.", "Specify Mission", ""),1,MAX_MESSAGE_LEN))
-		if(!input)
-			if(alert("Error, no mission set. Do you want to exit the setup process?",,"Yes","No")=="Yes")
-				return
+	var/input
```

Comment:
Having a variable name that clashes with a built-in, and then using both the var and the builtin on the same line is.. bold.
Rename to `mission`, probably

Reply:
Did this before in an older commit but scrapped all of this, readding it now